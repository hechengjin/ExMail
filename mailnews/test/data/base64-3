Content-Type: text/html
Content-Transfer-Encoding: base64

PGh0bWw+PGhlYWQ+VGhlIHRhZ3MgPGI+c2hvdWxkPC9iPiBiZSA8aT4gc3RyaXBwZWQgPC9pPiBpbiB0aGlzIGNhc2UhDQpObywgPHU+c2VyaW91c2x5LjwvdT48L2hlYWQ+PC9odG1sPg==
