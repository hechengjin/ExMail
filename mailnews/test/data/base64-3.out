The tags should be stripped in this case! No, seriously.